assets/helpers/page/page.js

```diff
@@ -49,6 +57,16 @@ function analyticsInitialisation(participations: Participations): void {
   ga.setDimension('experience', abTest.getVariantsAsString(participations));
   ga.trackPageview();
 
+  const dimensions:Dimensions = {
+    campaignCodeBusinessUnit: getQueryParameter('CMP_BUNIT') || undefined,
+    campaignCodeTeam: getQueryParameter('CMP_TU') || undefined,
+    experience: abTest.getVariantsAsString(participations),
+  };
+
+  if (!(doNotTrack() === '1' || doNotTrack() === 'yes')) {
```

Comment:
I would move this complexity inside the doNotTrack() method so you can just say `if(doNotTrack())` here